<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NMediaManager::NPreferences::NUi::CIgnoredPaths</class>
 <widget class="QWidget" name="NMediaManager::NPreferences::NUi::CIgnoredPaths">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>If enabled, these paths will be loaded in, but any media files found will be ignored for processing</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ignorePathNamesToIgnore">
     <property name="title">
      <string>Path Names to Ignore</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="1" column="1">
       <widget class="QToolButton" name="btnAddIgnorePathName">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../SABUtils/resources/SABUtils.qrc">
          <normaloff>:/SABUtilsResources/add.png</normaloff>:/SABUtilsResources/add.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="3">
       <widget class="QListView" name="pathNamesToIgnore"/>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="btnDelIgnorePathName">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../SABUtils/resources/SABUtils.qrc">
          <normaloff>:/SABUtilsResources/delete.png</normaloff>:/SABUtilsResources/delete.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer_9">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>174</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../SABUtils/resources/SABUtils.qrc"/>
 </resources>
 <connections/>
</ui>
